modules/ROOT/pages/_partials/gs-adding-users-proc.adoc

```diff
@@ -7,6 +7,16 @@ endif::[]
 [[sso]]
 = About Single Sign-On instances
 
+[NOTE]
+====
+You access your cluster using the credentials for the `customer-admin` user   received in the *Welcome* email. This user can:
+
+* access the OpenShift cluser with the permissions of the link:https://docs.openshift.com/dedicated/3/admin_guide/index.html#admin-guide-index-dedicated-admin-role[Dedicated cluster administrator Role]
+* access the services available from the Solution Explorer with administrator privileges.
```

Comment:
I'd expand upon explaining the "RHMI specific" bits this role gets:

- Read-only access to the Fuse Online namespace in OSD
- `admin` role in 3scale (https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.4/html/accounts/inviting-users-managing-rights#step_4_give_new_users_rights)
- On RH-SSO, this user has the `manage-user`, `view-realm` & `manage-identity-providers` roles applied (see https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.3/html/server_administration_guide/admin_permissions#realm_specific_roles)

cc'ing @maleck13 for sanity

Reply:
updated doc, but didn't include 3scale bit, cos i don't think that's the way we want them to create 3scale users, right?

Reply:
I think it's worthwhile mentioning that the `customer-admin` user has admin access to 3scale